static/bidder-info/iqzone.yaml

```diff
@@ -13,4 +13,10 @@ capabilities:
     - banner
     - video
     - native
-
+userSync:
+  redirect:
+    url: "https://cs.iqzone.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://cs.iqzone.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
```

Comment:
it works:
![image](https://github.com/user-attachments/assets/a47295bd-8106-4768-9fdc-35af1caecdd2)